### ISSUE TITLE
Pre-launch accessibility review

### ISSUE THREAD

**Issue by sarahmonster** [opentexts/opentexts#82]
Before we launch, we'll want to do a check to be sure the site confirms to accessibility standards.

@stuartlewis is there a particular accessibility standard we'd like to target? If not, I'd recommend we aim for WCAG 2.1 for a sensible baseline, but if there's a specific standard we need to meet, it would be good to know that in advance.

In addition to checking for standards, we'll want to test the site on a screen reader and using a keyboard only to be sure it's usable.

**Comment by stuartlewis** [comment]
Hi @sarahmonster,

Yes - we tend to aim for WCAG 2.1 AA.  We're covered by the new-ish Public Sector Accessibility regulations of 2018 in this regard: [https://www.legislation.gov.uk/uksi/2018/852/contents/made](https://www.legislation.gov.uk/uksi/2018/852/contents/made)

Thanks!

**Comment by sarahmonster** [comment]
Groovy—that's what I tend to aim for when building things out, so we should be on track there, and I'll just do a more in-depth review to catch any issues prior to launch.

**Comment by tofumatt** [comment]
Just did a quick accessibility audit at @sarahmonster's request: findings are good!

I went through the site with NVDA + Firefox; screen reader experience was great. Most things are good, but there's one issue:

<img width="633" alt="Screenshot 2020-10-04 at 20 56 59" src="https://user-images.githubusercontent.com/90871/95025795-3f287d80-0684-11eb-8978-9c47b8de9777.png">

The **download links** don't have `title` attributes or any screen reader text in them to indicate which article they're related to. This result in tabbing to the title of the article, then through three links that simply read "PDF", "IIF", and "Plain-text". But these links don't have any further context, so it's a bit tough to figure out what they're for. "Download PDF of <ARTICLE TITLE>" would be a good title or screenreader text here.

Otherwise great!

Keyboard navigation on the site is 💯.

No major issues found. 👍🏻
